src/frontend/Navigation/OnboardingStack.tsx

```diff
@@ -57,10 +58,8 @@ const MainStack = createSwitchNavigator(
 export const OnboardingStack = createStackNavigator(
   {
     Main: MainStack,
-    ProjectInviteModal: {
-      screen: ProjectInviteModal,
-      path: "ProjectInviteModal/:invite",
-    },
```

Comment:
Realized that the custom path wasn't necessary for this since React Navigation parses query params in deeplinks